docs/Software/InteractingWithClinica.md

```diff
@@ -133,9 +133,3 @@ You might also use `CNvsAD`, for instance, as group ID for a statistical group c
 --8<-- "snippets/converters_options.md"
 
 --8<-- "snippets/known_issues.md:matlab"
-
-
-## Support
```

Comment:
Shouldn't we link the corresponding snippet or am I missing something ?

Reply:
I added it to the list of automatically added snippets in the mkdocs.yml just like abbreviations.md so you get it everywhere :-) ! It is a bit overkill but you cannot miss it that way.

Reply:
OK, sounds good ! 👍